### PR TITLE
Fire Behaviour Calculator 

### PR DESCRIPTION
**Pull request by Sybrand**
Test here: https://wps-pr-1088.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator

- added prototype Fire Behaviour UI
- added prototype Fire Behaviour API
- modified defaults for CC (grass cure)
- added default mock for redis
- added R into github actions
- changed how automock for database works
- caching gdal in github actions for faster runs

### CODE REVIEW

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -12,13 +12,72 @@ jobs:
...
+          renv::restore()
+        shell: Rscript {0}
+
+      # - name: Restore environment
+      #   run: |
+      #     renv::consent(TRUE)
+      #     renv::restore()
+      #   shell: Rscript {0}
+
+      - name: Install R stuff
```
Comment (conbrad) [inline .github/workflows/integration.yml:76]: Duplicate with the previous step? (Install renv (api))

File: api/.pylintrc [hunk 2]
```diff
@@ -548,7 +548,9 @@ valid-metaclass-classmethod-first-arg=cls
 [DESIGN]
 
 # Maximum number of arguments for function / method.
-max-args=6
+# 2020/07/07:SS We very often have a lot of arguments, and that's just how it is. We're spending too much
+# time adding lint ignores.
+max-args=10
```
Comment (conbrad) [inline api/.pylintrc:553]: I'm not opposed to this but typically with long parameter lists there's groupings of parameters that make sense: https://refactoring.com/catalog/introduceParameterObject.html
(If there's consensus on this, let's not address it in this PR, wait for refactor task)

File: api/.pylintrc [hunk 3]
```diff
@@ -560,7 +562,7 @@ max-bool-expr=5
 max-branches=12
 
 # Maximum number of locals for function / method body.
-max-locals=15
+max-locals=20
```
Comment (conbrad) [inline api/.pylintrc:565]: Similar to the above, extracting functions/methods reduces the amount of locals: https://refactoring.com/catalog/extractFunction.html
(If there's consensus on this, let's not address it in this PR, wait for refactor task)

File: api/app/tests/fba/test_fba.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+from aiohttp import ClientSession
+import pytest
+import app.main
+from app.tests.common import default_mock_client_get
+from app.tests import load_json_file
+
+
+@pytest.mark.usefixtures('mock_jwt_decode')
+@scenario('test_fba.feature', 'Fire Behaviour Calculation',
+          example_converters=dict(request_json=load_json_file(__file__),
```
Comment (conbrad) [inline api/app/tests/fba/test_fba.py:16]: Nice

File: api/app/wildfire_one/wfwx_api.py [hunk 5]
```diff
@@ -59,7 +61,7 @@ async def get_stations_by_codes(station_codes: List[int]) -> List[WeatherStation
         header = await get_auth_header(session)
         stations = []
         # 1 day seems a reasonable period to cache stations for.
```
Comment (conbrad) [inline api/app/wildfire_one/wfwx_api.py:63]: Is this a week now?

File: api/app/wildfire_one/wfwx_api.py [hunk 6]
```diff
@@ -81,7 +83,7 @@ async def get_stations(session: ClientSession,
     """
     logger.info('Using WFWX to retrieve station list')
     # 1 day seems a reasonable period to cache stations for.
```
Comment (conbrad) [inline api/app/wildfire_one/wfwx_api.py:85]: Similar to above, stale comment?

File: api/app/wildfire_one/wfwx_api.py [hunk 7]
```diff
@@ -151,7 +153,7 @@ async def get_hourly_readings(
     # Create a list containing all the tasks to run in parallel.
     tasks = []
     # 1 day seems a reasonable period to cache stations for.
-    redis_station_cache_expiry: Final = int(config.get('REDIS_STATION_CACHE_EXPIRY', 86400))
+    redis_station_cache_expiry: Final = int(config.get('REDIS_STATION_CACHE_EXPIRY', 604800))
```
Comment (conbrad) [inline api/app/wildfire_one/wfwx_api.py:156]: Similar to above

**Review by conbrad — APPROVED** [review]

File: api/app/wildfire_one/wfwx_api.py (hunk 5 above)
Comment (Sybrand) [reply api/app/wildfire_one/wfwx_api.py:63]: Yes!

File: api/.pylintrc (hunk 2 above)
Comment (Sybrand) [reply api/.pylintrc:553]: I've added this comment to the tech debt ticket.

File: api/.pylintrc (hunk 3 above)
Comment (Sybrand) [reply api/.pylintrc:565]: I've added this comment to the tech debt ticket.

**Pull request merged**